django_fido/tests/test_models.py

```diff
@@ -122,10 +123,12 @@ def test_metadata_no_identification(self):
         authenticator = Authenticator(attestation_data=ATTESTATION_OBJECT)
         self.assertIsNone(authenticator.metadata)
 
-    def test_metadata_aaguid(self):
+    @patch('django_fido.models.crypto')
```

Comment:
What about freezing time to the point where the certificate was still valid instead of patching `django_fido.models.crypto`?

Reply:
OK, makes sense.